src/plugins/unified_field_list/public/index.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { UnifiedFieldListPlugin } from './plugin';
+
+export type {
+  FieldStatsResponse,
+  BucketedAggregation,
+  NumberStatsResult,
+  TopValuesResult,
+} from '../common/types';
+export type { FieldStatsProps, FieldStatsServices } from './components/field_stats';
+export { FieldStats } from './components/field_stats';
```

Comment:
The components are exported directly from the page load bundle which makes the bundle quite huge (>60kb) which will be downloaded on every load of Kibana even if the users doesn't use Discover/Lens. We should wrap them into a React.lazy wrapper to load them on demand.